libs/service-portal/settings/access-control/src/screens/Access/Access.tsx

```diff
@@ -209,153 +212,158 @@ function Access() {
+                        `${authDelegation?.to?.name} mun missa umboð fyrir eftirfarandi:` ||
+                        ''
+                      }
+                      scopes={scopes as ScopeTag[]}
+                      onClose={() => setCloseModalOpen(false)}
+                      onCloseButtonText={formatMessage({
+                        id:
+                          'sp.settings-access-control:access-remove-modal-cancel',
+                        defaultMessage: 'Hætta við',
+                      })}
+                      onSubmitColor="red"
+                      onSubmit={() => onDelete()}
+                      onSubmitButtonText={formatMessage({
+                        id:
+                          'sp.settings-access-control:access-remove-modal-confirm',
+                        defaultMessage: 'Já, ég vil eyða umboði',
+                      })}
+                    />
                   )}
-                </ModalBase>
+                </>
               )}
               <Button
                 size="small"
+                variant="text"
                 loading={updateLoading}
-                type="submit"
+                onClick={() => setSaveModalOpen(true)}
                 icon="checkmark"
               >
                 {formatMessage({
-                  id: 'service.portal.settings.accessControl:access-save',
+                  id: 'sp.settings-access-control:access-save',
                   defaultMessage: 'Vista aðgang',
                 })}
               </Button>
+              {saveModalOpen && (
+                <AccessModal
+                  id="access-save-modal"
+                  title={formatMessage({
+                    id: 'sp.settings-access-control:access-save-modal-content',
+                    defaultMessage: 'Ertu viss um að þú viljir veita umboðið?',
+                  })}
+                  //
+                  text={
+                    `${authDelegation?.to?.name}  mun fá umboð fyrir eftirfarandi:` ||
+                    ''
+                  }
+                  scopes={scopes as ScopeTag[]}
+                  onClose={() => setSaveModalOpen(false)}
+                  onCloseButtonText={formatMessage({
+                    id: 'sp.settings-access-control:access-save-modal-cancel',
+                    defaultMessage: 'Hætta við',
+                  })}
+                  onSubmitColor="blue"
+                  onSubmit={() => onSubmit()}
+                  onSubmitButtonText={formatMessage({
+                    id: 'sp.settings-access-control:access-save-modal-confirm',
+                    defaultMessage: 'Já, ég vil vista umboðið',
+                  })}
+                />
+              )}
             </Inline>
           </Box>
 
           <div className={styles.datePickerFix}>
-            <T.Table>
-              <T.Head>
-                <T.Row>
-                  <T.HeadData>
-                    <Text variant="small" color="blue600">
+            <Box>
```

Comment:
Nice! Can we now get rid of `<div className={styles.datePickerFix}>`?

Reply:
Yes def! ✅